### PR TITLE
Feat/usertfstats

### PR DESCRIPTION
**Pull request by emelalkim**

### CODE REVIEW

File: plugins/EpadDB.js [hunk 1]
```diff
@@ -13471,6 +13534,61 @@ async function epaddb(fastify, options, done) {
...
+          'admin'
+        );
+      }
+      reply.send('User TF statistics saved');
+    } catch (err) {
+      reply.send(new InternalError('Saving User TF statistics', err));
+    }
+  });
+
+  fastify.decorate('getUserTFStats', async (request, reply) => {
```
Comment (emelalkim) [inline plugins/EpadDB.js:13573]: add host

File: test/projectTest.js [hunk 2]
```diff
@@ -3345,6 +3377,157 @@ describe('Project Tests', () => {
...
+            expect(res.body.rows.length).to.be.eql(1);
+            done();
+          })
+          .catch((e) => {
+            done(e);
+          });
+      });
+
+      it('should trigger statistics calculation and sending ', (done) => {
+        chai
```
Comment (emelalkim) [inline test/projectTest.js:3458]: clean

File: config/test.js [hunk 3]
```diff
@@ -21,4 +21,5 @@ module.exports = {
   pollDW: 0,
   auditLog: true,
   versionAudit: true,
+  teachingProject: 'testaim3',
```
Comment (emelalkim) [inline config/test.js:24]: test with testaim

File: test/projectTest.js [hunk 4]
```diff
@@ -3345,6 +3377,157 @@ describe('Project Tests', () => {
...
+      it('should get user TF statistics ', (done) => {
+        chai
+          .request(`http://${process.env.host}:${process.env.port}`)
+          .get('/epads/usertfstats')
+          .query({ username: 'admin' })
+          .then((res) => {
+            expect(res.statusCode).to.equal(200);
+            expect(res.body).to.be.eql([
+              {
+                userId: 1,
```
Comment (emelalkim) [inline test/projectTest.js:3517]: get the year and month from current date

**Review by OzgeYurtsever — APPROVED** [review]

File: plugins/EpadDB.js (hunk 1 above)
Comment (emelalkim) [reply plugins/EpadDB.js:13573]: done

File: test/projectTest.js (hunk 4 above)
Comment (emelalkim) [reply test/projectTest.js:3517]: done

File: test/projectTest.js (hunk 2 above)
Comment (emelalkim) [reply test/projectTest.js:3458]: done

**Review by OzgeYurtsever — APPROVED** [review]

**Pull request merged**